modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/business-home-cover/index.js

```diff
@@ -139,46 +135,28 @@ getQuoteForm.onsubmit = function (event) {
 	const maxCharactersZIP = 5;
 
 	zipContainer.classList.remove('has-error');
-	productContainer.classList.remove('has-error');
 	zipErrorFeedback.innerText = '';
-	productErrorFeedback.innerText = '';
 
 	if (localStorage.getItem('raylife-back-to-edit')) {
 		localStorage.removeItem('raylife-back-to-edit');
 	}
 
-	const getProductName = (productId) => {
-		const options = product.options;
-
-		for (let i = 0; i < options.length; i++) {
-			if (options[i].value === productId) {
-				return options[i].label;
-			}
-		}
-	};
-
 	if (
 		!formProps.zip ||
-		formProps.zip.length !== maxCharactersZIP ||
-		!formProps.product
+		formProps.zip.length !== maxCharactersZIP 
+
 	) {
 		if (!formProps.zip || formProps.zip.length !== maxCharactersZIP) {
 			zipErrorFeedback.innerHTML =
 				'<span class="form-feedback-indicator"></span> Enter a valid 5 digit Zip';
 			zipContainer.classList.add('has-error');
 		}
-		if (!formProps.product) {
-			productErrorFeedback.innerHTML =
-				'<span class="form-feedback-indicator"></span> Please select a product';
-			productContainer.classList.add('has-error');
-		}
 	}
 	else {
 		localStorage.setItem(
 			'raylife-product',
 			JSON.stringify({
```

Comment:
`JSON.stringify(formProps)`

Could be passed directly

Reply:
Done